File: assignment 3/table.cpp
#include<iostream>
using namespace std;
class Table
{
    int i,num;
    public : void display()
    {
        cout<<"enter any number\n";
        cin>>num;
        for(i=1;i<=10;i++)
        {
            cout<<"\n"<<num*i;
        }
    }
};
int main()
{
    Table t1;
    t1.display();
}

File: JAVAAssignment 3/Table.java
import java.util.Scanner;
class Table
{
public static void main(String[] args)
{
int i,num;
System.out.println("Enter any number");
Scanner sc=new Scanner(System.in);
num=sc.nextInt();
for(i=1;i<=10;i++)
{
System.out.println(num*i);
}
}
}